acme/services.tf

```diff
@@ -0,0 +1,293 @@
+resource "pagerduty_service" "service_CRM" {
+  name                    = "CRM"
+  auto_resolve_timeout    = 14400
+  acknowledgement_timeout = 600
+  escalation_policy       = pagerduty_escalation_policy.it_crm.id
+  alert_creation          = "create_alerts_and_incidents"
+}
+
+resource "pagerduty_service" "service_web" {
+  name                    = "Web"
+  auto_resolve_timeout    = 14400
+  acknowledgement_timeout = 600
+  escalation_policy       = pagerduty_escalation_policy.it_web.id
+  alert_creation          = "create_alerts_and_incidents"
+}
+
+resource "pagerduty_service" "service_mainframe" {
+  name                    = "Mainframe"
+  auto_resolve_timeout    = 14400
+  acknowledgement_timeout = 600
+  escalation_policy       = pagerduty_escalation_policy.it_mainframe.id
+  alert_creation          = "create_alerts_and_incidents"
+}
+
+resource "pagerduty_service" "service_core" {
+  name                    = "Core IT Services"
+  auto_resolve_timeout    = 14400
+  acknowledgement_timeout = 600
+  escalation_policy       = pagerduty_escalation_policy.it_core.id
+  alert_creation          = "create_alerts_and_incidents"
+}
+
+
+## some services under IT core 
+
+resource "pagerduty_service" "service_dns" {
+  name                    = "Core DNS Services"
+  auto_resolve_timeout    = 14400
+  acknowledgement_timeout = 600
+  escalation_policy       = pagerduty_escalation_policy.it_core.id
+  alert_creation          = "create_alerts_and_incidents"
+}
+
+resource "pagerduty_service" "service_vmware" {
+  name                    = "Core VMware Services"
+  auto_resolve_timeout    = 14400
+  acknowledgement_timeout = 600
+  escalation_policy       = pagerduty_escalation_policy.it_core.id
+  alert_creation          = "create_alerts_and_incidents"
+}
+
+
+#how about some web services 
+resource "pagerduty_service" "service_customer_website" {
+  name                    = "acme.test"
+  auto_resolve_timeout    = 14400
+  acknowledgement_timeout = 600
+  escalation_policy       = pagerduty_escalation_policy.it_web.id
+  alert_creation          = "create_alerts_and_incidents"
+}
+
+resource "pagerduty_service" "service_ecommerce_website" {
+  name                    = "store.acme.test"
+  auto_resolve_timeout    = 14400
+  acknowledgement_timeout = 300 #gotta make that money money
+  escalation_policy       = pagerduty_escalation_policy.it_web.id
+  alert_creation          = "create_alerts_and_incidents"
+}
+
+resource "pagerduty_service" "service_customer_support_website" {
+  name                    = "support.acme.test"
+  auto_resolve_timeout    = 14400
+  acknowledgement_timeout = 600
+  escalation_policy       = pagerduty_escalation_policy.it_web.id
+  alert_creation          = "create_alerts_and_incidents"
+}
+
+
+#how about some business services 
```

Comment:
This file's pretty big -- should the business services be broken out?

Reply:
yea that seems like a good idea